src/main/java/org/apache/rocketmq/dashboard/service/impl/ConsumerServiceImpl.java

```diff
@@ -295,6 +301,9 @@ public boolean deleteSubGroup(DeleteSubGroupRequest deleteSubGroupRequest) {
             for (String brokerName : deleteSubGroupRequest.getBrokerNameList()) {
                 logger.info("addr={} groupName={}", clusterInfo.getBrokerAddrTable().get(brokerName).selectBrokerAddr(), deleteSubGroupRequest.getGroupName());
                 mqAdminExt.deleteSubscriptionGroup(clusterInfo.getBrokerAddrTable().get(brokerName).selectBrokerAddr(), deleteSubGroupRequest.getGroupName(), true);
+                // delete %RETRY%+Group and %DLQ%+Group in broker and namesrv
+                deleteDlqOrRetryTopic(MixAll.RETRY_GROUP_TOPIC_PREFIX + deleteSubGroupRequest.getGroupName(), brokerName, clusterInfo);
+                deleteDlqOrRetryTopic(MixAll.DLQ_GROUP_TOPIC_PREFIX + deleteSubGroupRequest.getGroupName(), brokerName, clusterInfo);
```

Comment:
deleteResources will be better than the existing method name.

Reply:
done